services/scabbard/src/cli/main.rs

```diff
@@ -0,0 +1,642 @@
+                .subcommand(
+                    SubCommand::with_name("create").args(&[
+                        Arg::with_name("namespace")
+                            .help("A global state address prefix (namespace)")
+                            .required(true),
+                        Arg::with_name("owner")
+                            .help("Owner of this namespace")
+                            .short("O")
+                            .long("owner")
+                            .required(true)
+                            .takes_value(true)
+                            .multiple(true),
+                    ]),
+                )
+                .subcommand(
+                    SubCommand::with_name("update").args(&[
+                        Arg::with_name("namespace")
+                            .help("A global state address prefix (namespace)")
+                            .required(true),
+                        Arg::with_name("owner")
+                            .help("Owner of this namespace")
+                            .short("O")
+                            .long("owner")
+                            .required(true)
+                            .takes_value(true)
+                            .multiple(true),
+                    ]),
+                )
+                .subcommand(
+                    SubCommand::with_name("delete").arg(
+                        Arg::with_name("namespace")
+                            .help("A global state address prefix (namespace)")
+                            .required(true),
+                    ),
+                ),
+            SubCommand::with_name("perm")
+                .about("Set or delete a Sabre namespace permission")
+                .args(&[
+                    Arg::with_name("namespace")
+                        .help("A global state address prefix (namespace)")
+                        .required(true),
+                    Arg::with_name("contract")
+                        .help("Name of the contract")
+                        .required(true)
+                        .conflicts_with("delete"),
+                    Arg::with_name("read")
+                        .help("Set read permission")
+                        .short("r")
+                        .long("read")
+                        .conflicts_with("delete"),
+                    Arg::with_name("write")
+                        .help("Set write permission")
+                        .short("w")
+                        .long("write")
+                        .conflicts_with("delete"),
+                    Arg::with_name("delete")
+                        .help("Remove all permissions")
+                        .short("d")
+                        .long("delete"),
+                ]),
+            SubCommand::with_name("cr")
+                .about("Create, update, or delete a Sabre contract registry")
+                .subcommand(
+                    SubCommand::with_name("create").args(&[
```

Comment:
Not all subcommands under cr, ns, and sp have usage help (as seen when doing 'cargo run -- cr --help'.).

Reply:
Added `about` for all subcommands.